deployment/docker/entrypoint.sh

```diff
@@ -30,6 +30,11 @@ echo 'Initialize project.'
 python manage.py collectstatic --clear --noinput
 python manage.py migrate
 
+# Command to create a superuser if it doesn't already exist
```

Comment:
Can you add the following
```
if [ -z "${DJANGO_SUPERUSER_USERNAME}" ]; then
    DJANGO_SUPERUSER_USERNAME=kartoza_admin
fi
if [ -z "${DJANGO_SUPERUSER_PASSWORD}" ]; then
    DJANGO_SUPERUSER_PASSWORD="Gs10w29k8*&"
fi
if [ -z "${DJANGO_SUPERUSER_EMAIL}" ]; then
    DJANGO_SUPERUSER_EMAIL=tinashe@kartoza.com
fi
```
This will ensure we have defaults and the dev team do not have to update anything